validator/Cargo.toml

```diff
@@ -22,6 +22,7 @@ validator_derive = { version = "0.14", path = "../validator_derive", optional =
 card-validate = { version = "2.2", optional = true }
 phonenumber = { version = "0.3", optional = true }
 unic-ucd-common = { version = "0.9", optional = true }
+indexmap = {version = "1", features = ["serde-1"], optional = true }
```

Comment:
why the serde feature?

Reply:
Essentially this allows you to use Serde for indexmap - without it your whole struct could be serializable but the indexmap wouldn't be (which of course wouldn't work)

Reply:
But we are implementing the trait on the base types no? I don't know the indexmap crate but it feels like we shouldn't need to use that feature

Reply:
The trait is implemented on the collection types AND indexmap/set, a very popular crate (ordered hash map/set - filling a stdlib gap). Indexmap code is only included if the feature is specified of course, so users don't pay the price for it unless they want it. If I didn't include the `serde-1` feature than the users wouldn't be able to serailize/deserailize any structs with an indexmap. I included the feature so that it would behave as expected to users (since you include `serde` - it is not a new dependency).

Reply:
Ah it's for passing the value to the error struct, I forgot about that requirement